site/docs/pages/index.mdx

```diff
@@ -351,6 +360,86 @@ setFilteredTokens(filteredTokens);
     </div>
   </aside>
 
+<main id="transaction" className="pt-[56px] flex flex-col gap-4 md:flex-row md:text-left text-center">
+  <a href="#transaction">
+    <h3 className="basis-1/2 py-[15px] text-4xl md:text-4xl">Transaction</h3>
+  </a>
+</main>
+<aside className="pb-6">
+  <p className="text-base md:text-base pb-[24px] md:text-left text-center">
+    Trigger onchain operations and sponsor them with [Paymaster](https://www.coinbase.com/developer-platform/products/paymaster).
+  </p>
+  <a href="/transaction/transaction" className="mx-auto md:m-0 flex justify-center items-center border border-indigo-400 text-indigo-400 font-bold leading-6 px-4 py-3 rounded-xl border-solid w-[210px]">
+    Start with Transaction
+  </a>
+  <div className="flex grow md:flex-row pt-[44px] flex-col">
+    <div className="w-[664px] max-w-full">
+```tsx
+<Transaction // [!code focus]
+  address={address}
+  chainId={84532}
```

Comment:
84532 does not mean much for Web2,

let's use instead `base.id` as it will be when importend from Viem